crates/consensus/beacon/src/engine/metrics.rs

```diff
@@ -13,6 +13,8 @@ pub(crate) struct EngineMetrics {
     pub(crate) forkchoice_updated_messages: Counter,
     /// The total count of new payload messages received.
     pub(crate) new_payload_messages: Counter,
+    /// The number of times the pruner was run.
+    pub(crate) pruner_runs: Counter,
```

Comment:
do we also want `pruned_blocks`?

Reply:
I think it will need to be a separate detailed prune metrics struct, as we can prune different number of blocks for different prune parts.